### PR TITLE
deleting tablets no longer leaves batteries

### LINKED ISSUE #24522: Deleting a mob leaves tablet batteries behind if there is a tablet in their bag
image

Seen this when people being incinerated in cremator, or being killed via SM

### PR DESCRIPTION
**Pull request by ghost**
:cl:
fix: To save costs, Nanotrasen has removed the emergency battery ejection systems in modular computers. We realized saving the batteries isn't really important.
/:cl:

fixes #24522

### CODE REVIEW

File: code/modules/modular_computers/hardware/battery_module.dm [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 /obj/item/computer_hardware/battery/handle_atom_del(atom/A)
 	if(A == battery)
-		try_eject(0, null, TRUE)
+		QDEL_NULL(battery)
```
Comment (SpaceManiac) [inline code/modules/modular_computers/hardware/battery_module.dm]: Isn't this called when `battery` is already being deleted?

Comment (Qustinnus) [reply code/modules/modular_computers/hardware/battery_module.dm]: battery is the battery inside of this item, which is actually a module for holding the batteries.

Comment (SpaceManiac) [reply code/modules/modular_computers/hardware/battery_module.dm]: guess where `handle_atom_del` is called from

Comment (Qustinnus) [reply code/modules/modular_computers/hardware/battery_module.dm]: hnng good point I'm brain farting. hold on I'll fix this

**Pull request merged**